drivers/hmis_csv_twenty_twenty_six/app/models/hmis_csv_twenty_twenty_six/exporter/export_concern.rb

```diff
@@ -179,6 +180,16 @@ def rounded_columns
       end
     end
 
+    # Remove forbidden characters from string fields and strip whitespace from the ends
+    def sanitize_string_fields(row)
+      row.each do |key, value|
+        next unless value.is_a?(String)
+
+        row[key] = value.gsub(/[<>\[\]{}]/, '')&.strip
```

Comment:
save nav operator not needed
```suggestion
        row[key] = value.gsub(/[<>\[\]{}]/, '').strip
```

Reply:
or maybe use transform values and also strip repeated white space
```ruby
    def sanitize_string_fields(row)
      row.transform_values! do |value|
        if value.is_a?(String)
          value.gsub(/[<>\[\]{}]/, '').gsub(/\s+/, ' ').strip
        else
          value
        end
      end
      row
    end
```